src/main/java/me/crespel/runtastic/RuntasticExportConverter.java

```diff
@@ -61,7 +69,25 @@ protected void doList(File path) throws FileNotFoundException, IOException {
 		SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
 		List<SportSession> sessions = converter.listSportSessions(path);
 		for (SportSession session : sessions) {
-			System.out.println(sdf.format(session.getStartTime()) + " - ID: " + session.getId() + ", type: " + session.getSportTypeId() + ", duration: " + session.getDuration() + "s");
+			System.out.println(sdf.format(session.getStartTime()) + " - ID: " + session.getId() + ", Sport Type: " + session.getSportTypeId() + ", duration: " + session.getDuration()/60000 + "min");
+		}
+	}
+
+	protected void doInfo(File path, String id) throws FileNotFoundException, IOException {
+		SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+		List<SportSession> sessions = converter.listSportSessions(path);
+		for (SportSession session : sessions) {
+			if ( session.getId().equalsIgnoreCase(id) ) {

```

Comment:
Since we're only interested in 1 session here, it would be more efficient to read and parse only that one, instead of the whole export folder. I would suggest adding a `getSportSession(File path, String id)` method to the `ExportConverter` class.

Reply:
I'll fix it for next commit.